pkg/cmd/extensions/manager.go

```diff
@@ -136,6 +136,8 @@ func (m *Manager) Install(cloneURL string, stdout, stderr io.Writer) error {
 	return externalCmd.Run()
 }
 
+var localExtensionUpgradeError = errors.New("local extensions can not be upgraded")
```

Comment:
I wonder should we keep calling these "local" extensions to the user or whether that could potentially be confusing? The way I see it, all extensions are technically _local_ after they've been installed locally. It's just that we've made a difference _when installing_ extensions from a "local" source (as opposed to from a remote repo).

Maybe we could call these "in development" extensions or something that would communicate their nature and why they shouldn't be attempted to be upgraded?

Reply:
I've been thinking of "local" as describing where updates come from and I think it's a very reasonable word to use. Updates either come from the network (remote/non-local) or the filesystem (local).